test/smoke/faucet.js

```diff
@@ -1,4 +1,23 @@
+const SSH = require('simple-ssh');
+const fs = require('fs');
+const utils = require('../testUtils');
+
+const getNetworkConfig = require('../../lib/test/getNetworkConfig');
+
+const networkConfig = getNetworkConfig();
+
 describe('Faucet', () => {
-  // getFunds
-  it('should respond positive balance');
+  networkConfig.inventory.web.hosts.forEach((nodeName) => {
+    it('should respond positive balance via insight and cli',
+      async () => {
+        const ssh = new SSH({
```

Comment:
Oh... faucet has RPC API which we can call with `fetch`: http://www.unibia.com/unibianet/developer/multifaucet-easy-setup-crypto-currency-faucet-supports-multiple-currencies